recipes/libtar/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "libtar" %}
+{% set version = "1.2.20" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://repo.or.cz/libtar.git/snapshot/0907a9034eaf2a57e8e4a9439f793f3f05d446cd.tar.gz
+  sha256: 4847207d878e79a4acbe32f096e57cd72c9507171953849e4d7eafe312418d95
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    # no idea, going with minor pin
```

Comment:
ABI Lab does not track this one. I believe we can request its addition but I don't know how to do that.

Reply:
Nice, thanks.